<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recipet</class>
 <widget class="QDialog" name="Recipet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt;
font: 28pt &quot;Snap ITC&quot;;
color: rgb(255, 255, 255);
font-weight: 900;
text-align: center;</string>
   </property>
   <property name="text">
    <string>RECEIPT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Store: Ain Shams Uni. SuperFcis Markert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Store Number: 19979</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Address: Abbassia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lobster</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>891</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>========================================================================================</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Click To Show Your RECEIPT</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 13px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Product Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Product Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Price</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>891</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>========================================================================================</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Click To Confirm Your Buy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>========================================================================================</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>========================================================================================</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>General Information </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>610</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 119, 255);
color: rgb(255, 255, 255);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Click To Back To Main Window</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>490</y>
     <width>269</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
      </property>
      <property name="text">
       <string>Amount Of Pay :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
      </property>
      <property name="text">
       <string>Order Number :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ordernum">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
      </property>
      <property name="text">
       <string>Order No.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
      </property>
      <property name="text">
       <string>Pay Price</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
